### DISCUSSION TITLE
Mode emoji is not found

### DISCUSSION THREAD

**Discussion by TAforever** [davatorium/rofi#1990] [(Possible) Issues] [answered]
I needed to build rofi-git and I noticed that the emoji mode was missing. Is this some kind of error or did you intentionally remove support for the emoji plugin?

**Comment by DaveDavenport** [comment, marked as answer]
I don't know what emoji plugin is, but you might need to recompile it against rofi-git.

**Reply by TAforever** [reply]
Yes, I also compiled rofi-emoji-git and everything worked again, thanks
